recipes/wget/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: wget
```

Comment:
Hm, as GNU Wget is about 15 years older and a few orders of magnitude more popular, it doesn’t feel right that this package should get the prefix and not the other one. But I agree it’s the easier way. I’ve renamed it.